docs/release-playbook.md

```diff
@@ -0,0 +1,203 @@
+# Bazel Release Playbook
+
+Status: Work in progress
+
+This is the guide to conducting a Bazel release. This is especially relevant for
+release managers, but will be of interest ti anyone who is curious about the
+release process.
+
+Each release has a tracking bug (see the
+[list](https://github.com/bazelbuild/bazel/issues?utf8=%E2%9C%93&q=label%3Arelease+)).
+The bug includes a "Target RC date". On that day, create a new release
+candidate.
+
+## Creating a new release candidate
+
+### Setup
+
+Do these steps once per release.
+
+*   Set up github ssh key if you haven't already.
+    *    https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
+*   Find baseline commit and cherrypicks
+    *    Check Bazel nightly build at
+         https://buildkite.com/bazel/bazel-with-downstream-projects-bazel. If
+         many downstream jobs are failing then this isnot a good baseline
+         commit. If only a few downstream jobs are failing and the issues are
+         known then this is a good baseline commit. Fixes for the known issues
+         should be cherry-picked, any remaining issues should become release
+         blockers.
+
+### Create a Candidate
+
+Create candidates with the release.sh script.
+
+*   If it's the first candidate for this version, run:
```

Comment:
nit. use ordered list? It might be easier to read or remember where I was

Reply:
Done.